org.osgi.test.cases.servlet/build.gradle

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Set the Java version of the launcher to Java 11
+ */
+testOSGi.javaLauncher = javaToolchains.launcherFor {
```

Comment:
I don't think we need this build.gradle file anymore since we now build and test on JDK 17.